src/commands/input.rs

```diff
@@ -0,0 +1,439 @@
+//! Implementation of the `input` command.
+
+use std::collections::HashSet;
+
+use anyhow::Result;
+use anyhow::bail;
+use clap::Parser;
+use serde_json::Map;
+use serde_json::Value;
+use wdl::analysis::types::CallKind;
+use wdl::ast::AstNode;
+use wdl::ast::AstToken;
+use wdl::ast::SyntaxKind;
+use wdl::ast::v1::Expr;
+use wdl::ast::v1::InputSection;
+use wdl::ast::v1::LiteralExpr;
+use wdl::ast::v1::Type;
```

Comment:
this uses a lot from `ast`. Is it possible to use some more from `analysis` to simplify the code?

To be clear, I don't know. Just a hunch 🤷‍♀️

Reply:
Agreed. If there is a way to get the relevant information from `analysis`, that'd be ideal. Admittedly, I started with `ast` and switched to `analysis` later (I haven't used it previously). It's possible I missed features in the `analysis` crate.